neptune/management/internal/api.py

```diff
@@ -115,9 +110,15 @@ def get_project_list(api_token: Optional[str] = None) -> List[str]:
 
 
 @with_api_exceptions_handler
+def _get_projects(params, api_token: Optional[str] = None) -> List:
+    verify_type("api_token", api_token, (str, type(None)))
```

Comment:
I think we should verify as soon as possible (in user-exposed methods) and in private assume that parameters are correct